src/app/screens/Discover/websites.json

```diff
@@ -159,6 +159,12 @@
         "subtitle": "Lightning-fast UI for Nostr",
         "logo": "https://cdn.getalby-assets.com/alby-extension-website-screen/primat-logo_fire-409917ad.svg",
         "url": "https://primal.net/"
+      },
+      {
+        "title": "Broadcstr",
```

Comment:
Can you please move this item so it's alphabetically sorted? Thanks!

Reply:
Done!